.github/workflows/tests.yml

```diff
@@ -19,8 +19,8 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', 'pypy3']
-        isort: [4.3.21, 5.9.3]
-        flake8: [3.9.2, 4.0.1]
+        isort: [4.3.21, 5.10.1]
+        flake8: [3.9.2, 4.0.1, 5.0.1]
```

Comment:
flake8 has bumped to 5.0.4, can you update this target and test again, please?

Reply:
@nhymxu would you mind applying this suggestion?

Reply:
@GriceTurrble @gforcada 

I bump version to 5.0.4